dio/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: dio
-version: 5.3.1
+version: 5.3.2
```

Comment:
You can revert the version bump here and in the changelog, we do this automatically when releasing.

Reply:
So I just set it as unreleased correct?

Reply:
Yes, in the changelog.